packages/cardhost/app/components/card-manipulator.js

```diff
@@ -150,4 +222,44 @@ export default class CardManipulator extends Component {
   delete() {
     this.deleteCard.perform();
   }
+
+  @action
+  beginDragging(fieldComponent, mousedownEvent) {
+    let dragState = {
+      usingKeyboard: false,
+      initialPointerX: mousedownEvent.x,
+      initialPointerY: mousedownEvent.y,
+      latestPointerX: mousedownEvent.x,
+      latestPointerY: mousedownEvent.y
+    };
+
+    window.addEventListener('dragend', () => set(this, 'isDragging', false));
```

Comment:
where do we remove these event listeners? I think this will mean that you just accumulate new event listeners every time dragging begins. and then when dragging ends every accumulated event listener will be executed (so this function could be executed dozens of times for each dragend). If the reason we are doing this is for closure, then let's use the new octane trio of action helpers/modifiers (`(on)`,`(action)`, and `(fn)`). I've commented in the template what this should look like.

I think the best practice here is that instead of `window.addEventListner` you should favor using an `{{on}}`, because when you do, ember will take are of all the removing of the event listeners for you which is something that you really need to add here if you are gonna wire the event listeners by hand.

Reply:
this is a copy-paste of pre-Octane code, will update it.